docs/course_materials/linux_intro.html

```diff
@@ -0,0 +1,367 @@
+
+  <link rel="stylesheet" href="reveal/css/reveal.css">
+  <link rel="stylesheet" href="media/presentations.css" id="theme">
+  <link rel="stylesheet" href="reveal/lib/css/zenburn.css">
+
+  <!-- Printing and PDF exports -->
+  <script>
+    var link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.type = 'text/css';
+    link.href = window.location.search.match(/print-pdf/gi) ? 'reveal/css/print/pdf.css' : 'reveal/css/print/paper.css';
+    document.getElementsByTagName('head')[0].appendChild(link);
+  </script>
+
+  <style>
+    .reveal .slide spre {
+      width: 100%
+    }
+  </style>
+</head>
+
+<body>
+
+  <div class="reveal">
+    <div class="slides">
+
+      <section data-background-size="cover" data-background-image="media/rose_background.jpg">
+        <h1 class="slide_main_title"><b>Linux Introduction</b></h1>
+      </section>
+      <section>
+        <section>
+          <h3><b>Open a terminal window</b></h3>
+          <ul>
+            <li>Open the search menu:</li>
+            <ul>
+              <li>Press the "Windows" key</li>
+              <li>Or move the mouse to the upper left of the screen</li>
+            </ul>
+            <li>Type "terminal" in the search bar</li>
+            <li>Click on the icon named "Terminal"</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$
+            </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>Getting more info on commands</b></h3>
+          <ul>
+            <li>man</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$man pwd
+PWD(1)
+
+NAME
+       pwd - print name of current/working directory
+
+SYNOPSIS
+       pwd [OPTION]...
+        </code></pre>
+          <ul>
+            <li>--help</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$pwd --help
+pwd: pwd [-LP]
+    Print the name of the current working directory.
+
+    Options:
+      -L	print the value of $PWD
+...
+      </code></pre>
+        </section>
+      </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>pwd</b></h3>
+          <ul>
+            <li>Prints the current directory name</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$pwd
+/home/student
+$
+        </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>mkdir</b></h3>
+          <ul>
+            <li>Creates a directory in the current directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir my_test
+$
+        </code></pre>
+          <ul>
+            <li>Creates a directory in a specific directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir /home/student/my_test
```

Comment:
If we're on /home/student already, the 'my_test' folder was created on line 109.
Maybe we should change its name to "my_second_test" or "my_test2" ?

Reply:
Done